generators/bootstrap-application-base/utils.mjs

```diff
@@ -57,7 +56,8 @@ export function createUserEntity(customUserData = {}, application) {
   loadRequiredConfigIntoEntity(user, this.jhipsterConfigWithDefaults);
 
   const oauth2 = user.authenticationType === OAUTH2;
-  const userIdType = oauth2 || user.databaseType !== SQL ? TYPE_STRING : getPkType(this.jhipsterConfig, user.databaseType);
+  // If oauth2 or databaseType is cassandra, force type string, otherwise keep undefined for later processing.
```

Comment:
These utils class should disappear: is it possible to extracts these methods on the go when modified?